src/Data/Uniques/claw.lua

```diff
@@ -112,54 +113,44 @@ Variant: Pre 3.19.0
 Variant: Current
 Requires Level 37, 53 Dex, 77 Int
 Implicits: 3
-{variant:1}+15 Life gained for each Enemy hit by Attacks
-{variant:2}+21 Life gained for each Enemy hit by Attacks
-{variant:3,4}+25 Life gained for each Enemy hit by Attacks
+{variant:1}Gain 15 Life per Enemy Hit by Attacks
+{variant:2}Gain 21 Life per Enemy Hit by Attacks
+{variant:3,4}Gain 25 Life per Enemy Hit by Attacks
 {variant:1,2,3}(8-12)% increased Cast Speed
 {variant:4}(15-20)% increased Cast Speed
 +(30-40) to maximum Mana
-{variant:1,2,3}+(5-8) Life gained for each Enemy hit by your Spells
-{variant:4}+(15-20) Life gained for each Enemy hit by your Spells
-{variant:1}6% increased Spell Damage per 5% Block Chance
-{variant:2,3,4}8% increased Spell Damage per 5% Block Chance
+{variant:1,2,3}Gain (5-8) Life per Enemy Hit with Spells
+{variant:4}Gain (15-20) Life per Enemy Hit with Spells
+{variant:1}6% increased Spell Damage per 5% Chance to Block Attack Damage
+{variant:2,3,4}8% increased Spell Damage per 5% Chance to Block Attack Damage
 ]],[[
 Essentia Sanguis
-Eye Gouger
+{variant:1,2,3}Eye Gouger
+{variant:4,5,6}Vaal Claw
 Variant: Pre 1.3.0
 Variant: Pre 2.0.0
 Variant: Pre 2.6.0
-Requires Level 64, 113 Dex, 113 Int
+Variant: Pre 3.11.0
+Variant: Pre 3.20.0
+Variant: Current
 Implicits: 3
 {variant:1,2}0.6% of Physical Attack Damage Leeched as Life
-{variant:3}+31 Life gained for each Enemy hit by Attacks
-{variant:1}10% additional Chance to Block while Dual Wielding Claws
-{variant:2,3}8% additional Chance to Block while Dual Wielding Claws
+{variant:3}Gain 31 Life per Enemy Hit by Attacks
+{variant:4,5}2% of Physical Attack Damage Leeched as Life
+{variant:1}+10% Chance to Block Attack Damage while Dual Wielding Claws
+{variant:2,3,4,5}+8% Chance to Block Attack Damage while Dual Wielding Claws
 {variant:1}(80-120)% increased Physical Damage
-{variant:2,3}(140-180)% increased Physical Damage
+{variant:2,3,4,5}(140-180)% increased Physical Damage
 {variant:1}Adds 1 to 50 Lightning Damage
 {variant:2,3}Adds 1 to 80 Lightning Damage
+{variant:4,5}Adds 1 to 200 Lightning Damage
+{variant:6}Adds 1 to (500-600) Lightning Damage
```

Comment:
Patch notes have been updated, new version adds 1 to 600-700 lightning

Reply:
Updated patch notes for essentia and divinaurs